### PR TITLE
add Size to readme

### LINKED ISSUE #232: document image size calculations and restrictions
This can be added to the readme file.

### PR DESCRIPTION
**Pull request by jeffvance**
Add Size() info and restrictions to readme.
Fixes #232

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,57 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
...
+- _any.iso.tar[.gz|.xz]_
+
+The `Size` function does **not** yet support the following formats:
+- _any.gz_
+
+- _any.xz_
+
+- certain raw, unititialized iso formats
+
+In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (even with compression) then the size of the endpoint can be determined. If the endpoint is a simple an unformatted raw file then its size cannot (yet) be returned.
```
Comment (screeley44) [inline README.md]: s/is a simple an/is a simple and

File: README.md [hunk 2]
```diff
@@ -135,6 +135,57 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
...
+
+And additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
```
Comment (screeley44) [inline README.md]: `And additional caveat` Or `An additional caveat`

File: README.md [hunk 3]
```diff
@@ -135,6 +135,57 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
...
+   ...
+   "kubevirt.io/containerized-data-importer/pkg/lib/size"
+   ...
+   size, err := size.Size(sourceFileEp, "", "")
+   if err == nil {
+      ... // have a valid size
+   }
+```
+##### Limitations:
+The `Size` function currenly supports the following file formats:
```
Comment (copejon) [inline README.md]: sp - currently

File: README.md [hunk 4]
```diff
@@ -135,6 +135,57 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
...
+The `Size` function does **not** yet support the following formats:
+- _any.gz_
+
+- _any.xz_
+
+- certain raw, unititialized iso formats
+
+In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (even with compression) then the size of the endpoint can be determined. If the endpoint is an unformatted raw file, not compressed, not archived, then its size cannot (yet) be returned.
+
+And additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
```
Comment (copejon) [inline README.md]: sp - An ...

**Pull request merged**